Icebnb/Icebnb/MapViewWidget.swift

```diff
@@ -10,14 +10,17 @@ import UIKit
 import MapKit
 import InstantSearch
 import InstantSearchCore
+import AlgoliaSearch
```

Comment:
Would be best to hide AlgoliaSearch from developers (unless they really want to dig in deep). All "high level" functionalities should be surfaced from InstantSearch (stating the obvious I know...)

Reply:
Done